.github/workflows/conda-envs/requirements-test-oldest.txt

```diff
@@ -1,5 +1,5 @@
 ase==3.16.0
```

Comment:
We need to name this file something besides `requirements` -- Dependabot triggers version updates on all files named `requirements*.txt`. We don't want Dependabot to update our oldest dependencies. You could name it `test-oldest.txt` or `oldest-reqs.txt` or something like that.

(I can't find the reference that lists the pattern anymore -- I believe I had to search Dependabot's source code the first time I set this up. I can't find a reference to this in the PRs where I added this file, but I thought I had left a comment somewhere for historical reasons...)